src/metabolic_ninja/tasks.py

```diff
@@ -152,7 +152,7 @@ def find_pathways(product, model, max_predictions, source):
     return predictor.run(
         product,
         max_predictions=max_predictions,
-        timeout=60,
+        timeout=120,  # seconds
```

Comment:
My original intention was to permit up to one hour. I think it should be more than two minutes. Maybe 30?
```suggestion
        timeout=30 * 60,  # seconds
```

Reply:
It is 2 min per pathway, which is enough.